src/pages/dar_application/ProgressReportApplication.tsx

```diff
@@ -10,15 +11,17 @@ import SubmitProgressReport from 'src/pages/progress_reports/SubmitProgressRepor
 import {DataUseAcknowledgements} from 'src/pages/dar_application/DataUseAcknowlegements';
 import {translateDataUseRestrictionsFromDataUseArray} from 'src/libs/dataUseTranslation';
 import {validatePRFormData} from 'src/utils/darFormUtils';
-import {FormValidationState} from 'src/pages/dar_application/FormValidationState';
+import {FormValidationState, ValidationError} from 'src/pages/dar_application/FormValidationState';
 
 type ProgressReportApplicationProps = {
-    dar: DataAccessRequest, // corresponds either to the parent DAR for a new application or an existing readonly progress report
-    datasets: Dataset[],
-    readOnlyMode: boolean
+  readonly dar: DataAccessRequest, // corresponds either to the parent DAR for a new application or an existing readonly progress report
+  readonly datasets: Dataset[],
+  readonly readOnlyMode?: boolean
+  readonly location?: Location
+  readonly researcher: DuosUser
```

Comment:
Do these need commas for consistency?

Reply:
I see a couple different approaches which all seem to work which is confusing. I see no separators, comma separators, and semi-colon separators. Looking at some docs, I think semi-colon is the expectation.

Reply:
> Does our linter not do anything here?

Unfortunately, I'm not seeing any warnings in my setup - I think we need to look into adding something for this.